src/main/java/commands/inputParser.java

```diff
@@ -0,0 +1,115 @@
+package commands;
+
+import exceptions.AragornException;
+
+public class inputParser {
+    private String[] splitInput = new String[3];
+
+    public inputParser(String userInput, String commandType) throws AragornException {
```

Comment:
this function is quite long (30+ LOC). consider abstracting out the code into multiple functions 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods